repl.py

```diff
@@ -303,18 +305,20 @@ def check_prev(arg):
             remaining.append(arg)
 
     if len(remaining) == 0:
```

Comment:
This prints a command-line usage error ("Missing argument") when something on the filesystem is missing.  This could be confusing to users.

Another way to write it:

```
if len(remaining) == 0:
    auth_file = default_auth_json_path()
    if not os.path.isfile(auth_file):
        err.write("Couldn't find default auth.json file at {}.\n".format(devq(auth_file)))
        err.write("Run with \"--help\" for more information.\n");
        sys.exit(1)
elif len(remaining) == 1:
    auth_file = remaining[0]
else:
    err.write("Expecting at most one non-option argument, got {}: {}"
              .format(len(remaining), devql(remaining)))
    err.write("Run with \"--help\" for more information.\n")
    sys.exit(1)
```

Reply:
👍